pandas/core/frame.py

```diff
@@ -553,8 +553,17 @@ def __init__(
         index: Optional[Axes] = None,
         columns: Optional[Axes] = None,
         dtype: Optional[Dtype] = None,
-        copy: bool = False,
+        copy: Optional[bool] = None,
```

Comment:
Can you update the docstring above for this change?

Reply:
good idea, will do.